src/components/organisms/CardCompanySelect/CardCompanySelect.tsx

```diff
@@ -0,0 +1,38 @@
+import { Dispatch, SetStateAction } from 'react';
+import useOverlay from '@/hooks/useOverlay';
+import { BottomSheet } from '@components/molecules/BottomSheet/BottomSheet';
+import CardCompany from '@components/organisms/CardCompany/CardCompany';
+import { CardCompany as CardCompanyType } from '@/type/cardType';
+import { FormValues } from '@/type/formType';
+
+interface CardCompanySelect {
+  onChange: Dispatch<SetStateAction<FormValues>>;
+  onAutoFocus?: () => void;
+}
+
+export default function CardCompanySelect({
+  onChange,
+  onAutoFocus,
+}: CardCompanySelect) {
+  const { close: closeBottomSheet } = useOverlay();
+
+  const handleCompanyClick = (company: CardCompanyType) => {
+    onChange((prevCardInfo) => ({
+      ...prevCardInfo,
+      cardCompany: company,
+    }));
+
+    onAutoFocus ? onAutoFocus() : null;
```

Comment:
```suggestion
onAufoFocus?.()

if(onAutoFocus) {
onAutoFocus()
}

onAutoFocus ? onAutoFocus() : undefiend
```

해당 케이스에서 null의 사용은 조금 어색한 감이 있다고 생각해요

일반적인 옵셔널 체이닝에서 해당 값은 undefined로 추론됩니다!